test/endecode_test.c

```diff
@@ -1525,6 +1547,8 @@ int setup_tests(void)
 # endif
 #endif
 #ifndef OPENSSL_NO_EC
+        if (!is_fips_lt_3_5)
+            ADD_TEST(ec_encode_to_data_multi);
```

Comment:
Why is this needed? This is a bug fix and should go into all the active branches, IMO.

Reply:
@slontis Is the is_fips_lt_3_5 check needed? Encoders are in the base provider, not fips.
Also this should be merged to all the branches.